src/test/nodestore/Database_test.cpp

```diff
@@ -156,32 +159,52 @@ class Database_test : public TestBase
                 BEAST_EXPECT(areBatchesEqual (batch, copy));
             }
         }
-    }
-
-    //--------------------------------------------------------------------------
 
-    void runBackendTests (std::int64_t const seedValue)
-    {
-        testNodeStore ("nudb", true, seedValue);
-
-    #if RIPPLE_ROCKSDB_AVAILABLE
-        testNodeStore ("rocksdb", true, seedValue);
-    #endif
-    }
+        {
```

Comment:
nit - it don't think this outer block serves any purpose.

Reply:
👍 removed